### PR TITLE
docs: Move contributors to CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by acurat**
Move contributors to CONTRIBUTORS.md as the list is getting long in CONTRIBUTING.md

https://github.com/gitpoint/git-point/issues/359

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Contributors
+
+Thank you to all the people who have already contributed to git-point!
```
Comment (lex111) [inline CONTRIBUTORS.md]: > git-point

Let's rename to GitPoint?

Comment (vitalkanev) [reply CONTRIBUTORS.md]: It looks like an actual repository name, that PR author written here...

Comment (andrewda) [inline CONTRIBUTORS.md]: As @lex111 said, the official name is "GitPoint", so I'd say we should probably use that. I'm actually not sure why there's a dash in the repository name 😂

Comment (acurat) [reply CONTRIBUTORS.md]: Done!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -79,19 +79,7 @@ Anyone can file an expense. If the expense makes sense for the development of th
...
-| [<img src="https://avatars0.githubusercontent.com/u/18398761?v=4" width="100px;"/><br /><sub>Vitaliy Kanev</sub>](https://github.com/vitalkanev)<br />[📖](https://github.com/gitpoint/git-point/commits?author=vitalkanev "Documentation") | [<img src="https://avatars3.githubusercontent.com/u/3691490?v=4" width="100px;"/><br /><sub>Peter Dave Hello</sub>](https://www.peterdavehello.org/)<br />[📖](https://github.com/gitpoint/git-point/commits?author=PeterDaveHello "Documentation") [🔧](#tool-PeterDaveHello "Tools") | [<img src="https://avatars3.githubusercontent.com/u/10849025?v=4" width="100px;"/><br /><sub>Ernoff</sub>](https://github.com/Ernoff)<br /> |
-<!-- ALL-CONTRIBUTORS-LIST:END -->
-
+Checkout all the awesome people who have contributed [here](./CONTRIBUTORS.md)
```
Comment (lex111) [inline CONTRIBUTING.md]: Missing dot.

File: CONTRIBUTORS.md (hunk 1 above)
Comment (housseindjirdeh) [reply CONTRIBUTORS.md]: @lex111 I always use `kebab-case` for repository names, but I really don't know why I didn't just name it `gitpoint` :thinking:

File: .all-contributorsrc [hunk 3]
```diff
@@ -381,6 +382,15 @@
         "code",
         "bug"
       ]
+    },
+    {
+      "login": "acurat",
```
Comment (housseindjirdeh) [inline .all-contributorsrc]: :tada:

**Pull request merged**